addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -396,27 +396,20 @@ def _import_fill_invoice_allowance_charge(self, tree, invoice_form, journal, qty
 
     def _import_fill_invoice_down_payment(self, invoice_form, prepaid_node, qty_factor):
         """
-        Creates a down payment line on the invoice at import if prepaid_node (TotalPrepaidAmount in CII,
-        PrepaidAmount in UBL) exists.
+        Log a message in the chatter at import if prepaid_node (TotalPrepaidAmount in CII, PrepaidAmount in UBL) exists.
         qty_factor -1 if the xml is labelled as an invoice but has negative amounts -> conversion into a credit note
         needed, so we need this multiplier. Otherwise, qty_factor is 1.
         """
         if prepaid_node is not None and float(prepaid_node.text) != 0:
-            # create a section
-            with invoice_form.invoice_line_ids.new() as invoice_line_form:
-                invoice_line_form.sequence = 998
-                invoice_line_form.display_type = 'line_section'
-                invoice_line_form.name = _("Down Payments")
-                invoice_line_form.price_unit = 0
-                invoice_line_form.quantity = 0
-                invoice_line_form.account_id = self.env['account.account']
-            # create the line with the down payment
-            with invoice_form.invoice_line_ids.new() as invoice_line_form:
-                invoice_line_form.sequence = 999
-                invoice_line_form.name = _("Down Payment")
-                invoice_line_form.price_unit = float(prepaid_node.text)
-                invoice_line_form.quantity = qty_factor * -1
-                invoice_line_form.tax_ids.clear()
+            amount = float(prepaid_node.text) * qty_factor
+            move_type = dict(self.env['account.move']._fields['move_type'].selection)[invoice_form.move_type]
+            return [
+                _("%(amount)s %(currency)s are indicated as prepaid for this %(move)s.",
```

Comment:
what if the currency has a prefix instead of suffix?
`formatLang` perhaps?

Reply:
100.0001 $ are indicated as prepaid for this vendor credit note?

I'm not sure about the language here...
maybe "A prepayment of $100.00 has been indicated" or something like that?

Reply:
or "A payment of 64.62 € was detected"